webclient/src/containers/Login/Login.tsx

```diff
@@ -1,5 +1,4 @@
-// eslint-disable-next-line
-import React, { useState, useCallback } from "react";
```

Comment:
Likewise, i think we still need to import React here, otherwise we end up with some sort of build/compile error that i'm struggling to remember.

Reply:
As of 17 `React` no longer needs to be in scope simply to use JSX.  [Here's an article from the react blog](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html) if you're interested in a deeper dive on this.

Removing `eslint-disable-next-line` _and_ the `React` import resulted in no build errors for me locally.  Do the build errors manifest upstream in CI somewhere?

Reply:
I've already confirmed that our versions of CRA and TypeScript support the new JSX transform.

Reply:
Oh ok, didnt know that had happened. Lets run with it and deal with any errors if they happen.